src/databricks/labs/ucx/install.py

```diff
@@ -778,6 +799,11 @@ def _get_ext_hms_conf_from_policy(cluster_policy):
                 spark_conf_dict[key[11:]] = cluster_policy[key]["value"]
         return instance_profile, spark_conf_dict
 
+    def _write_mapping_conf_file(self, default_catalog: str):
+        mapping = "element,target,notes\n"
+        mapping += f"<root>,{default_catalog},Default Mapping created by UCX Install\n"
```

Comment:
I don't really understand the scope of this PR, why would we write a mapping file when the user can only enter the default catalog name ?